datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/recommended_monitors.py

```diff
@@ -0,0 +1,100 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import json
+import os
+
+import click
+
+from ....utils import file_exists, read_file
+from ...constants import get_root
+from ...utils import get_valid_integrations, load_manifest
+from ..console import CONTEXT_SETTINGS, abort, echo_failure, echo_info, echo_success
+
+REQUIRED_ATTRIBUTES = {'name', 'type', 'query', 'message', 'tags', 'options', 'recommended_monitor_metadata'}
+
+
+@click.command(
+    'recommended-monitors',
+    context_settings=CONTEXT_SETTINGS,
+    short_help='Validate recommended monitor definition JSON files',
+)
+def recommended_monitors():
+    """Validate all recommended monitors definition files."""
+    root = get_root()
+    echo_info("Validating all recommended monitors...")
+    failed_checks = 0
+    ok_checks = 0
+
+    for check_name in sorted(get_valid_integrations()):
+        display_queue = []
+        file_failed = False
+        manifest = load_manifest(check_name)
+        monitors_relative_locations = manifest.get('assets', {}).get('monitors', {}).values()
+        for monitor_relative_location in monitors_relative_locations:
+
+            monitor_file = os.path.join(root, check_name, *monitor_relative_location.split('/'))
+            if not file_exists(monitor_file):
```

Comment:
This logic is similar to what I used in saved views validation. If it's not too much trouble we could abstract this out to a `get_assets_from_manifest(asset)` function:
 https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/tooling/utils.py#L243

Reply:
I'm going to open a separate PR for the refactor and incorporate

Reply:
https://github.com/DataDog/integrations-core/pull/7282